billy_metadata/wy.py

```diff
@@ -33,6 +33,13 @@
             'end_year': 2018,
             'sessions': ['2017'],
         },
+        {
+            'name': '2018-2019',
```

Comment:
The `2018` session is part of the `2017-2018` term, not a standalone `term` in and of itself.